docs/docs/examples/core-components/point-attractor.md

```diff
@@ -0,0 +1,217 @@
+---
+sidebar_position: 3
+title: Point Attractors
+---
+
+import pointAttractor from './assets/point-attractor.png'
+import pointAttractorExample from './assets/point-attractor-example.gif'
+
+# Point Attractors
+
+This page shows how Dynamical Systems, and in particular Point Attractor components, can be used to generate dynamic
+motions in AICA Studio. As described [here](../../concepts/robotics-concepts/motion-generation.md), the motion of a
+Point Attractor DS is always directed toward a point in space, the attractor. The core components come with two
+implementations of a Point Attractor, the `Signal Point Attractor` whose inputs and output are poses and a twist in
+Cartesian space and the `Signal Joint Point Attractor` which acts on signals in joint space.
+
+## Signal Point Attractor (Cartesian space)
+
+### Interfaces
+
+The component has three inputs and one output, all of type `cartesian_state`;
+
+- Input pose: The current state of the Dynamical System, e.g. the pose that should be determined to calculate the motion
+  towards the attractor.
+- Attractor pose: The attractor of the Dynamical System, e.g. the pose that the motion should end up in.
```

Comment:
```suggestion
- Attractor pose: The attractor of the Dynamical System, e.g. the pose towards which the motion guides the robot.
```

sounds a bit better to me, without saying this is inaccurate. So up to what you prefer